website/source/docs/provisioners/ansible.html.md

```diff
@@ -50,6 +50,12 @@ Optional Parameters:
 - `command` (string) - The command to invoke ansible.
    Defaults to `ansible-playbook`.
 
+- `inventory_file` (string) - The file or directory to use.
+
+- `inventory_path` (string) - The path where to place dynamically generated inventory by packer.
+  Useful when you set `inventory_file` as directory. Defaults empty string, but internally
+  equals to `os.TempDir`.
```

Comment:
This is to much detail for the user. It should say something like defaults to `$TEMPDIR`.